### PR TITLE
Add basic support for binary-mode CloudEvent messages

### PR DESCRIPTION
**Pull request by kevmoo**
Add conformance tests for http and cloud events

Progress towards https://github.com/GoogleCloudPlatform/functions-framework-dart/issues/1
Fixes https://github.com/GoogleCloudPlatform/functions-framework-dart/issues/31

### CODE REVIEW

**Review by subfuzion — APPROVED** [review]
This is awesome!

File: functions_framework_builder/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 0.2.1-dev
+
+- many things...
```
Comment (subfuzion) [inline functions_framework_builder/CHANGELOG.md:3]: Maybe just mention this adds basic support for binary cloudevent messages...?

Comment (kevmoo) [reply functions_framework_builder/CHANGELOG.md:3]: Will do

**Comment by kevmoo** [comment]
@subfuzion – I pushed more changes, so I need another approval

**Review by subfuzion — APPROVED** [review]
👍

**Pull request merged**